src/index.css

```diff
@@ -1,17 +1,22 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
 
-Button {
-  margin: 20px;
+.sunflower-logo {
+  width: 75%;
+}
+
+.header-container {
+  display: flex;
+  justify-content: center;
```

Comment:
Please create a CSS file for the Header separately and add it in the component directory with the Header component
```
components
|____ Header
         |____ index.tsx
         |____Header.css
```

Reply:
Got it!